src/pool/srv_pool.c

```diff
@@ -1243,7 +1243,7 @@ read_db_for_stepping_up(struct pool_svc *svc, struct pool_buf **map_buf,
 			 */
 			D_DEBUG(DB_MD, DF_UUID": new db\n",
 				DP_UUID(svc->ps_uuid));
-			rc = +DER_UNINIT;
```

Comment:
huh? is this a typo of + instead of -  in the original code too?

Reply:
Odd.  I seem to remember we do have one place in the code where a positive error code is used for some reason where there's a clear reason why negative wouldn't work.  Like you say not a regression;